assets/schema_input.json

```diff
@@ -15,18 +15,21 @@
             },
             "source": {
                 "type": "string",
+                "pattern": "^\\S+$",
                 "errorMessage": "Source of genome must be provided and cannot contain spaces",
                 "meta": ["source"]
             },
             "source_vcf": {
                 "type": "string",
-                "errorMessage": "Source of the VCF file must be provided and cannot contain spaces",
+                "pattern": "^\\S+$",
+                "errorMessage": "Source of the VCF, cannot contain spaces",
```

Comment:
not blocking here. but does source here mean something else than in source_version?

Reply:
yeah, I need better identifier for the fields, I'll fix it in a separate PR

Reply:
https://github.com/nf-core/references/issues/65